manager/controllers/package_systems.go

```diff
@@ -30,6 +30,8 @@ type PackageSystemItemCommon struct {
 	AvailableEVRA string         `json:"available_evra" csv:"available_evra" query:"spkg.latest_evra" gorm:"column:available_evra"`
 	Updatable     bool           `json:"updatable" csv:"updatable" query:"(update_status(spkg.update_data) = 'Installable')" gorm:"column:updatable"`
 	Tags          SystemTagsList `json:"tags" csv:"tags" query:"null" gorm:"-"`
+	OSAttributes
+	BaselineIDAttr
```

Comment:
This change will add those fields also to V2 API, please add it only in `PackageSystemItemV3`

Reply:
Moved, thanks for noticing.